server/src/main/java/org/opensearch/extensions/ExtensionsOrchestrator.java

```diff
@@ -179,9 +192,15 @@ public void initializeServicesAndRestHandler(
             transportService,
             REQUEST_EXTENSION_UPDATE_SETTINGS
         );
+        this.client = client;
+        this.extensionActions = new ExtensionActions(extensionIdMap, transportService, client);
         registerRequestHandler();
     }
 
+    public ExtensionActionResponse handleTransportRequest(ExtensionActionRequest request) throws InterruptedException {
```

Comment:
`handleActionRequest`?

Reply:
Sure, if you like that better.

Reply:
I like `handleTransportAction` to differentiate it from `handleRestAction` etc.

Reply:
I'll leave it as `handleTransportAction` for now.